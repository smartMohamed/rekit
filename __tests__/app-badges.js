import React from 'react'
import Enzyme, {mount} from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import AppBadge from '@/components/app-badge'

Enzyme.configure({ adapter: new Adapter() })

const props = {
  name: 'apple',
  url: 'https://app.adjust.com/p39urb',
  label: 'Laden im'
}

describe('App Badge component', () => {
  it('Should render component and match snapshot', () => {
    const wrapper = mount(<AppBadge { ...props } />)
    expect(wrapper.html()).toMatchSnapshot()
  })
})