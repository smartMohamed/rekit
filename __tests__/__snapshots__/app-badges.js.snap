// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Badge component Should render component and match snapshot 1`] = `"<a href=\\"https://app.adjust.com/p39urb\\" class=\\"fk-app-btn \\" target=\\"_blank\\" rel=\\"noopener noreferrer\\">Laden im</a>"`;
